### DISCUSSION TITLE
Smooth scrolling : experiment available in pre-release

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#2397] [Announcements]
In pre-release (https://www.sumatrapdfreader.org/prerelease) you can enable smooth scrolling experiment as described in https://www.sumatrapdfreader.org/settings/settings by setting `SmoothScrolling = true`.

I'm not sure if that's an improvement so please test and comment.

The difference is: when scrolling by a page, instead of jumping right to there, it scrolls to desired position by smaller increments.

**Comment by tylvn** [comment]
Thank you very much for your contribution, this feature does improve the experience when browsing PDF files, which has as smooth as browsing on Microsoft Edge, but there is a delay for a while compared to Microsoft Edge. When I scrolled, it would take a while before the PDF file would follow. Especially when I drag the scroll bar with the mouse, the delay is most noticeable.

**Comment by gitgrub** [comment]
To be honest, I do not like smooth scrolling at all, tend to turn it off in every program.
While it looks cool, I want to get to the desired position as fast as possible.

I think if we want smooth scrolling, it should be a system-wide setting, fine grained configurable, as it depends on the hardware and the user/users eyes. Then any program would simply turn it on or off. The scrolling behavior should not differ between applications.

**Reply by tylvn** [reply]
Yeah, it is the best way for smooth scrolling to become a system-wide setting, but Microsoft just seems to have the grained configurable on Microsoft Edge.
